src/ctia/graphql_service.clj

```diff
@@ -1,27 +0,0 @@
-(ns ctia.graphql-service
-  (:require [ctia.graphql-service-core :as core]
-            [ctia.store-service :as store-svc]
-            [puppetlabs.trapperkeeper.core :as tk]
-            [puppetlabs.trapperkeeper.services :refer [service-context]]))
-
-(defprotocol GraphQLService
-  (get-graphql [this] "Returns an instance of graphql.GraphQL"))
-
-(tk/defservice graphql-service
-  GraphQLService
-  [ConfigService
-   StoreService
-   GraphQLNamedTypeRegistryService
-   IEncryption]
```

Comment:
As we discussed today, now there's a cycle between `CTIAHTTPServerService` and `GraphQLService`, since `CTIAHTTPServerService` already uses `GraphQLService`, and now `GraphQLService` uses `CTIAHTTPServerService` to get the current port instead of `ConfigService`.

Alternatively we could create a third service that just handles the dynamic configuration of the port, which `CTIAHTTPServerService` would set and `GraphQLService` would read.